src/Grids/grid_utils.jl

```diff
@@ -393,6 +385,14 @@ function pop_flat_elements(tup, topo)
     return Tuple(new_tup)
 end
 
+"""
+    min_number_or_array(var)
+
+Return the minimum value of `var`.
+"""
+@inline min_number_or_array(var) = var
+@inline min_number_or_array(var::AbstractVector) = minimum(parent(var))
```

Comment:
I don't understand why this is needed. `minimum` seems to do the same?

```Julia
julia> minimum([1, 2, 3])
1

julia> minimum(1)
1
```

Reply:
Is the `parent()` important here?

Reply:
I didn't write that code (I just moved it), so I don't know why `parent()` is important. It also seems to me like we can use just `minimum`. I can change that line and we see if the tests pass!

Reply:
```Julia
julia> using CUDA, OffsetArrays

julia> a = 2.0
 2.0

julia> b = CUDA.randn(4, 4)
4×4 CuArray{Float32, 2, CUDA.Mem.DeviceBuffer}:
 -0.0972989  -0.0783534   0.167717   1.1295
  0.0279738   0.682576    0.238343  -0.158799
  1.0389     -0.49114     1.74628   -0.892839
  0.682562   -1.47588    -0.7601     1.26731

julia> c = OffsetArray(b, -3, -2)
4×4 OffsetArray(::CuArray{Float32, 2, CUDA.Mem.DeviceBuffer}, -2:1, -1:2) with eltype Float32 with indices -2:1×-1:2:
┌ Warning: Performing scalar indexing on task Task (runnable) @0x00007fa4c226d5a0.
│ Invocation of getindex resulted in scalar indexing of a GPU array.
│ This is typically caused by calling an iterating implementation of a method.
│ Such implementations *do not* execute on the GPU, but very slowly on the CPU,
│ and therefore are only permitted from the REPL for prototyping purposes.
│ If you did intend to index this array, annotate the caller with @allowscalar.
└ @ GPUArraysCore ~/.julia/packages/GPUArraysCore/B3xv7/src/GPUArraysCore.jl:103
 -0.0972989  -0.0783534   0.167717   1.1295
  0.0279738   0.682576    0.238343  -0.158799
  1.0389     -0.49114     1.74628   -0.892839
  0.682562   -1.47588    -0.7601     1.26731

julia> minimum(a)
2.0

julia> minimum(b)
-1.4758823f0

julia> minimum(c)
┌ Warning: Performing scalar indexing on task Task (runnable) @0x00007fa674cec010.
│ Invocation of getindex resulted in scalar indexing of a GPU array.
│ This is typically caused by calling an iterating implementation of a method.
│ Such implementations *do not* execute on the GPU, but very slowly on the CPU,
│ and therefore are only permitted from the REPL for prototyping purposes.
│ If you did intend to index this array, annotate the caller with @allowscalar.
└ @ GPUArraysCore ~/.julia/packages/GPUArraysCore/B3xv7/src/GPUArraysCore.jl:103
-1.4758823f0
```

Reply:
I believe `min_number_or_array` is redundant. I'm nuking it.